src/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Kanit&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/6e8f4ae605.js" crossorigin="anonymous"></script>
+    <title>Document</title>
+</head>
+<body>
+    <header class="nav">
+      <a href="#" class="logo"><img id="img" alt="Logo"></a>
+        <div class="nav-list">
+          <a class="listLink" href="#" id="list-link">Home</a>
+            <span class="divider">&#124</span>
+          <a class="addNewLink" href="#" id="add-new-link"></a>
+        </div>
```

Comment:
Consider making the navigation consistent with the given mock up, because you are currently using `=` for the count but the mockup uses `()`, and also the counter link should come first as seen in the [mockup](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/images/Home.png). It seen uneccssary but this would help your ability to pay attention to tiny details which is an irrefutable skill for a frontend developer.

<img width="282" alt="ch" src="https://user-images.githubusercontent.com/84930355/233544037-114aa811-0985-41d2-918d-ecc811e5430b.PNG">

Reply:
Hello 
Thank you for your feedback 
I have implemented the required changes